lib/src/arithmetic/signals/floating_point_logics/floating_point_logic.dart

```diff
@@ -67,6 +67,17 @@ class FloatingPoint extends LogicStructure {
   FloatingPointValuePopulator valuePopulator() => FloatingPointValue.populator(
       exponentWidth: exponent.width, mantissaWidth: mantissa.width);
 
+  /// Return true if the J-bit is not explicitly represented in the mantissa
```

Comment:
Why are these `Const` instead of `bool`?

Reply:
also why do we have both?  can't one just always the be the opposite of the other?